bodhi/server/services/updates.py

```diff
@@ -308,8 +309,15 @@ def query_updates(request):
         query = query.filter(Update.date_pushed < pushed_before)
 
     releases = data.get('releases')
+    old_updates = data.get('old_updates')
     if releases is not None:
         query = query.filter(or_(*[Update.release == r for r in releases]))
+    elif old_updates is None:
```

Comment:
This will default to a backwards-incompatible change in the API - let's make the new behavior opt-in instead of opt-out so the API is preserved. Perhaps the new parameter should be called "exclude_old_updates" or something like that.

Also, ```old_updates``` is always ```None``` so it is currently not possible to opt out and the UI button does not seem to work as a result. I think this is because we need to add this new parameter to the scemas here so it shows up in ```data```:

https://github.com/fedora-infra/bodhi/blob/2.11.0/bodhi/server/schemas.py#L388-L558

Reply:
> This will default to a backwards-incompatible change in the API

Do we have a way to know when something is asked via the API or via the UI? I kinda really think displaying non-actionable updates in the UI isn't really useful.

> UI button does not seem to work as a result.

This is odd, it worked for me yesterday when I tried it :/
Indeed, adding it to ``schemas.py`` makes it work as well now, odd.